agent_api_rest/postman/ssi-agent.postman_collection.json

```diff
@@ -104,6 +107,60 @@
 					},
 					"response": []
 				},
+				{
+					"name": "Thuiswinkel Waarborg Credential",
```

Comment:
Since this is only part of the examples inside Postman and not part of the code itself, I would be okay with leaving it as is if you don't want to refactor again.

Reply:
Still think it is better to use ACME Corp so I've updated the Collection :+1: